README.md

```diff
@@ -41,22 +118,23 @@ It is required to setup an AWS account and access the AWS IoT Console for runnin
 
 -  [Setup an AWS account](https://docs.aws.amazon.com/iot/latest/developerguide/iot-console-signin.html).
 
--  [Sign-in to the AWS IoT Console](https://docs.aws.amazon.com/iot/latest/developerguide/iot-console-signin.html) after setting up the AWS account.
+-  [Sign-in to the AWS IoT Console](https://aws.amazon.com/console/) after setting up the AWS account.
 
 
 
 *Note: If using the Provisioning library, a fleet provisioning template, a provisioning claim, IoT policies and IAM policies need to be setup for the AWS account. Complete the steps to setup your device and AWS IoT account outlined [here](https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html#use-claim).*
 
 
 
-### Configuring the mutual auth demos
 
-- You can pass the following configuration settings as command line options in order to run the mutual auth demos: 
+### Configuring the MQTT mutual auth demo
```

Comment:
```suggestion
### Configuring the MQTT mutual auth demos
```

Reply:
There is only a single mutual auth demo, isn't it?

Reply:
There is a mutual auth demo for http as well.

Reply:
But this mentions `**MQTT** mutual auth"

Reply:
Yea. Sorry, my bad. I didn't notice MQTT in there. I will update the suggestion